docs/introduction/introduction.md

```diff
@@ -264,11 +258,31 @@ const uniqueIdentifier = url.split('/').slice(-2).join('/');
 ### Test run 2
 We'll add our first data to the `pageFunction` and carry out a test run to see that everything works as expected.
 
-{{#code}}test-run-2.js{{/code}}
+```js
```

Comment:
Is this correct? I think the `{{#code}}test-run-2.js{{/code}}` was there so that we can automatically test the code examples if they still work.

Reply:
Yep, sorry about that!